docs/activities/Development_Guides.mdx

```diff
@@ -609,6 +609,7 @@ Under the hood we utilize Cloudflare Workers, which brings some restrictions, ou
 #### Supported Ports
 
 See the full list of supported ports for network traffic, as listed in [Cloudflare's Docs](https://developers.cloudflare.com/fundamentals/reference/network-ports/#network-ports-compatible-with-cloudflares-proxy).
```

Comment:
what is this line useful for then?

Reply:
Good callout. I think I need to be more specific, here. So an activity can make a requests to one of these supported ports, i.e. `https://<app_id>.discordsays.com`. Separately, the url mapping can route the final request to a specific port, but... due to the known issue listed below, the port will be removed and the original port of the original request will be maintained